NebulaWorld/Chat/Commands/WhoCommandHandler.cs

```diff
@@ -0,0 +1,99 @@
+﻿using NebulaAPI;
+using NebulaModel.Packets.Players;
+using NebulaWorld.MonoBehaviours.Local;
+using System.Text;
+
+namespace NebulaWorld.Chat.Commands
+{
+    public class WhoCommandHandler : IChatCommandHandler
+    {
+        private readonly IPlayerManager _playerManager;
+
+        public WhoCommandHandler()
+        {
+            _playerManager = Multiplayer.Session.Network.PlayerManager;
```

Comment:
I’m not sure getting multiplayer session here is safe. This is called in static constructor and the value could be null.